argilla/tests/integration/test_create_datasets.py

```diff
@@ -70,8 +68,7 @@ def test_create_multiple_dataset_with_same_settings(self, client: Argilla):
             assert schema["question"].name == "question"
             assert schema["question"].values == [1, 2, 3, 4, 5]
 
-    def test_create_dataset_from_existing_dataset(self, client: Argilla):
-        dataset_name = f"test_dataset_{uuid.uuid4()}"
+    def test_create_dataset_from_existing_dataset(self, client: Argilla, dataset_name: str):
```

Comment:
should we check if task_distribution is included correctly during such operations?